nunaliit2-multimedia/src/main/java/ca/carleton/gcrc/olkit/multimedia/file/SystemFile.java

```diff
@@ -91,16 +92,19 @@ static public SystemFile getSystemFile(File file) throws Exception {
 			List<String> tokens = new Vector<String>();
 			tokens.add("file");
 			tokens.add("-bnk");
-			tokens.add("--mime");
```

Comment:
Removed as the --mime changes the output so that the file matchers (both for mp3 and m4a) will not have the needed content provided to match. It appears to me that this was not working for mp3 before my changes.